src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2484,13 +2484,13 @@
     <value>Open parent folder of searched item</value>
   </data>
   <data name="RefreshItemsDescription" xml:space="preserve">
-    <value>Refresh page contents</value>
+    <value>Reload page contents</value>
```

Comment:
Either term is fine, but we've already used the term 'Refresh' throughout the codebase so we might as well keep it as it is. Another reason to keep 'Refresh' is to avoid the need to retranslate.

Reply:
No problem 🙂